src/rt-app.c

```diff
@@ -1133,6 +1156,7 @@ void *thread_body(void *arg)
 		struct timespec t_diff, t_rel_start;
 
 		set_thread_affinity(data, &pdata->cpu_data);
+		set_thread_membind(data, &pdata->numa_data);
```

Comment:
minor: Could you keep the same ordering  here and above, ie:
set_thread_priority
set_thread_membind
set_thread_taskgroup

Reply:
Sure, will do.